plugin/trino-iceberg/pom.xml

```diff
@@ -238,6 +257,40 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Why add these dependencies here? Are we trying to override the version?

Reply:
The dependency checker flags these as `Used undeclared dependencies`, but they're direct dependencies of `iceberg-core`.

Reply:
The dependency checker flags these as Used undeclared dependencies, but they're direct dependencies of iceberg-core.